ui/helpers/utils/transactions.util.js

```diff
@@ -212,7 +212,7 @@ export function getTransactionTypeTitle(t, type, nativeCurrency = 'ETH') {
       return t('transferFrom');
     }
     case TRANSACTION_TYPES.TOKEN_METHOD_APPROVE: {
-      return t('approve');
+      return t('approveLimit');
```

Comment:
This change seems unrelated to the snap install page. A few of the locale changes related to approving seem unrelated as well. Merge conflict resolution mistake perhaps?

Reply:
It's a fix for snap-confirm.js having wrong button text

Reply:
Ahh I see, OK thanks!